ckanext/harvest/harvesters/base.py

```diff
@@ -21,16 +21,29 @@
 from ckan.plugins.core import SingletonPlugin, implements
 from ckanext.harvest.interfaces import IHarvester
 
+if p.toolkit.check_ckan_version(min_version='2.3'):
+    from ckan.lib.munge import munge_tag
```

Comment:
Is there a reason not to use a `try` ... `except ImportError` block for this? This would work, even if a future version of CKAN does no longer include `munge_tag`.